1-installation/index.md

```diff
@@ -35,82 +48,158 @@ Hint: don't worry if you make a typo when writing commands into the <span class=
 
-	$ pwd
+<div class="os-switch">
+<span class="osx">
 
-`pwd` is short for "**p**rint **w**orking **d**irectory". If your username were `codecat`(remember the `whoami` command?), the result should look something like this:
+<div class="highlighter-rouge"><pre class="highlight"><code>$ pwd</code></pre></div>
+<code class="highlighter-rouge">pwd</code> is short for "<strong>p</strong>rint <strong>w</strong>orking <strong>d</strong>irectory".
 
-	$ pwd
-	/Users/codecat
+If your username were <code class="highlighter-rouge">codecat</code>(remember the <code class="highlighter-rouge">whoami</code> command?), the result should look something like this:
 
-Now, to see a list of files and folders inside the current folder, use the following command:
+<div class="highlighter-rouge"><pre class="highlight"><code>$ pwd
+</code><code>/Users/codecat</code></pre></div>
 
-<div class="os-switch">
-<div class="osx">
-<pre><code>$ ls</code></pre>
-</div>
-<div class="wins">
-<pre><code>&gt; dir</code></pre>
-</div>
+</span>
+<span class="wins">
+
+<div class="highlighter-rouge"><pre class="highlight"><code>> cd</code></pre></div>
+
+<code class="highlighter-rouge">cd</code> is short for "<strong>c</strong>urrent <strong>d</strong>irectory".
+
+If your username were <code class="highlighter-rouge">codecat</code>(remember the <code class="highlighter-rouge">whoami</code> command?), the result should look something like this:
+
+<div class="highlighter-rouge"><pre class="highlight"><code>> cd
+</code><code>C:\Users\codecat</code></pre></div>
+
+</span>
 </div>
 
-`ls` is a command that "**l**i**s**ts" files in the current folder. The result should look something like this:
+Now, to see a list of files and folders inside the current folder, use the following command:
 
-	$ ls
-	Applications
-	Desktop
-	Documents
-	Downloads
-	...
+<div class="os-switch">
+<span class="osx">
+<div class="highlighter-rouge"><pre class="highlight"><code>$ ls</code></pre></div>
+
+<code class="highlighter-rouge">ls</code> is a command that "<strong>l</strong>i<strong>s</strong>ts" files in the current folder. The result should look something like this:
+
+<div class="highlighter-rouge"><pre class="highlight"><code>
+$ ls
+Applications
+Desktop
+Documents
+Downloads
+...
+</code></pre></div>
+</span>
+
+<span class="wins">
+<div class="highlighter-rouge"><pre class="highlight"><code>> dir</code></pre></div>
+
+<code class="highlighter-rouge">dir</code> is a command used for file and "<strong>dir</strong>ectory" listing. The result should look something like this:
+
+<div class="highlighter-rouge"><pre class="highlight"><code>
+Directory of C:\Users\codecat
+
+16.10.2015  09:32  .
+16.10.2015  09:32  ..
+16.10.2015  09:30  Desktop
+16.10.2015  09:30  Documents
+16.10.2015  11:06  Downloads
+03.05.2015  17:17  Favorites
+...
+</code></pre></div>
+</span>
+</div>
 
 Now, let's move into the `Desktop` folder:
 
 	$ cd Desktop
```

Comment:
I think lines like this should also get a Windows version, so it's not confusing to see $ and > changing throughout the tutorial.